testsuite/features/step_definitions/command_steps.rb

```diff
@@ -493,8 +493,8 @@
 end
 
 When(/^the server stops mocking an IPMI host$/) do
-  $server.run('kill $(pidof ipmi_sim)')
-  $server.run('kill $(pidof -x fake_ipmi_host.sh)')
+  $server.run('kill $(pgrep ipmi_sim)')
+  $server.run('if pgrep fake_ipmi_host.sh &> /dev/null; then kill $(pgrep fake_ipmi_host.sh); else :; fi')
```

Comment:
`pgrep fake_ipmi_host.sh &> /dev/null && kill $(pgrep fake_ipmi_host.sh)` maybe?

Reply:
In my local tests the second `pgrep` doesn't find the process; in such a case, with your form,
run will raise (the form would exit with code 1).
It might be that when using `pgrep`, we are killing everything in the first line; then we would not need the second.
I tried to be defensive, so that my form always return 0 exit code.

Reply:
Indeed, we don't need a subshell: `pkill` is also available, we can just `pkill process-foo`.